src/TheReplacement.PTA.Common/Models/NpcModel.cs

```diff
@@ -28,5 +28,41 @@ public class NpcModel : IPerson, IDocument
 
         /// <inheritdoc />
         public IEnumerable<string> Feats { get; set; }
+
+        /// <inheritdoc />
```

Comment:
Your inheritdocs aren't really doing anything since those properties aren't implementations of the interfaces. You'll need to put a summary xmldoc on these properties explaining what they do. The easy part is that the TrainerModel class has all of these new properties already, so you should be able to copy directly from there.

Reply:
added xmldocs specified